src/Error/ErrorLogger.php

```diff
@@ -153,7 +153,7 @@ protected function getMessage(Throwable $exception, bool $isPrevious = false, bo
         }
 
         if ($includeTrace) {
-            $trace = Debugger::formatTrace($exception, ['format' => 'shortPoints']);
+            $trace = Debugger::formatTrace($exception, ['format' => Configure::read('Error.traceFormat', 'shortPoints')]);
```

Comment:
```suggestion
            $trace = Debugger::formatTrace(
                $exception,
                ['format' => Configure::read('Error.traceFormat', 'shortPoints')],
            );
```

This should fix the CS error.

Reply:
We should raise the limit to 140 imo 
120 is kinda short in modern days